Sources/NavigationStack/NavigationStack.swift

```diff
@@ -77,7 +77,7 @@ public class NavigationStack: ObservableObject {
 
         mutating func push(_ element: ViewElement) {
             if indexForView(withId: element.id) != nil {
-                return
+                fatalError("NavigationStackView: you are trying to push a view with an identifier that already exists on the navigation stack. Identifier: \"\(element.id)\"")
```

Comment:
```suggestion
                fatalError("Duplicated view identifier: \"\(element.id)\". You are trying to push a view with an identifier that already exists on the navigation stack.")
```
Moved the important bit at the beginning of the line. Moreover fatalError will print file and line number, no need to specify where the error comes from.